Dockerfile

```diff
@@ -186,12 +149,23 @@ RUN make entrypoints
 RUN --mount=type=cache,target=/root/.cache \
     --mount=type=cache,target=/var/lib/localstack/cache \
     source .venv/bin/activate && \
+    python -m localstack.cli.lpm install java --version 11 && \
```

Comment:
q / nit: If we can manage to avoid setting the JAVA_HOME, I think we should avoid setting the version here. We should just install the default version (which is 11 now anyways, but we would dynamically move to new defaults in the future).

Reply:
As mentioned in another comment, the ideal situation is where all LPM packages have explicit Java dependencies. This will let us get rid of the above default installation of Java 11 altogether. This is planned for later iterations.